Dataset: ./data/cats_dogs_full/train and ./data/cats_dogs_full/test
Train dataloader: 624 batches
Test dataloader: 157 batches
--------Epoch: 0
-Training...
Batch: 0/624
/home/danielcarias/.local/lib/python3.10/site-packages/PIL/TiffImagePlugin.py:870: UserWarning: Truncated File Read
  warnings.warn(str(msg))
Batch: 400/624
Per batch -> Train Loss: 0.6930574774742126 | Train Accuracy: 50.58%
-Testing...
Batch: 0/157
Batch: 50/157
Batch: 100/157
Batch: 150/157
Per batch -> Test Loss: 0.6930896639823914 | Test Accuracy: 50.10%
--------Epoch: 1
-Training...
Batch: 0/624
/home/danielcarias/.local/lib/python3.10/site-packages/PIL/TiffImagePlugin.py:870: UserWarning: Truncated File Read
  warnings.warn(str(msg))
Batch: 400/624
Per batch -> Train Loss: 0.6930068135261536 | Train Accuracy: 51.10%
-Testing...
Batch: 0/157
Batch: 50/157
Batch: 100/157
Batch: 150/157
Per batch -> Test Loss: 0.6929327845573425 | Test Accuracy: 52.69%
--------Epoch: 2
-Training...
Batch: 0/624
Batch: 400/624
Per batch -> Train Loss: 0.6928046941757202 | Train Accuracy: 51.36%
-Testing...
Batch: 0/157
Batch: 50/157
Batch: 100/157
Batch: 150/157
Per batch -> Test Loss: 0.6928436756134033 | Test Accuracy: 50.50%
--------Epoch: 3
-Training...
Batch: 0/624
Batch: 400/624
Per batch -> Train Loss: 0.6927219033241272 | Train Accuracy: 51.48%
-Testing...
Batch: 0/157
Batch: 50/157
Batch: 100/157
Batch: 150/157
Per batch -> Test Loss: 0.6927474737167358 | Test Accuracy: 53.24%
--------Epoch: 4
-Training...
Batch: 0/624
Batch: 400/624
Per batch -> Train Loss: 0.6925075650215149 | Train Accuracy: 53.31%
-Testing...
Batch: 0/157
Batch: 50/157
Batch: 100/157
Batch: 150/157
Per batch -> Test Loss: 0.6926092505455017 | Test Accuracy: 52.53%
--------Epoch: 5
-Training...
Batch: 0/624
Batch: 400/624
Traceback (most recent call last):
  File "/home/danielcarias/Documents/Estudo NN/Estudo/CNN/first_project/CNN.py", line 154, in <module>
    train_loss, train_acc = train_model(model, train_dataloader, loss_fn, optimizer)
  File "/home/danielcarias/Documents/Estudo NN/Estudo/CNN/first_project/CNN.py", line 48, in train_model
    for batch, (X, y) in enumerate(data_loader):
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torchvision/datasets/folder.py", line 229, in __getitem__
    sample = self.loader(path)
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torchvision/datasets/folder.py", line 268, in default_loader
    return pil_loader(path)
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torchvision/datasets/folder.py", line 248, in pil_loader
    return img.convert("RGB")
  File "/home/danielcarias/.local/lib/python3.10/site-packages/PIL/Image.py", line 922, in convert
    self.load()
  File "/home/danielcarias/.local/lib/python3.10/site-packages/PIL/ImageFile.py", line 291, in load
    n, err_code = decoder.decode(b)
KeyboardInterrupt
Traceback (most recent call last):
  File "/home/danielcarias/Documents/Estudo NN/Estudo/CNN/first_project/CNN.py", line 154, in <module>
    train_loss, train_acc = train_model(model, train_dataloader, loss_fn, optimizer)
  File "/home/danielcarias/Documents/Estudo NN/Estudo/CNN/first_project/CNN.py", line 48, in train_model
    for batch, (X, y) in enumerate(data_loader):
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torchvision/datasets/folder.py", line 229, in __getitem__
    sample = self.loader(path)
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torchvision/datasets/folder.py", line 268, in default_loader
    return pil_loader(path)
  File "/home/danielcarias/.local/lib/python3.10/site-packages/torchvision/datasets/folder.py", line 248, in pil_loader
    return img.convert("RGB")
  File "/home/danielcarias/.local/lib/python3.10/site-packages/PIL/Image.py", line 922, in convert
    self.load()
  File "/home/danielcarias/.local/lib/python3.10/site-packages/PIL/ImageFile.py", line 291, in load
    n, err_code = decoder.decode(b)
KeyboardInterrupt